lib/elixir/lib/calendar.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Calendar do
```

Comment:
@josevalim I think we should rename `Calendar` to something different, and use `Calendar` to define the behaviour for implementing another calendar. Unless you prefer it be both the entry point module for all date/time things + the behaviour definition.